src/utils/dex/types.js

```diff
@@ -0,0 +1,13 @@
+const { typeName } = require('../../../__gamedata');
+
+function getTypeName(typeId = 0) {
+  return typeName.labelDataArray[typeId].wordDataArray[0].str;
```

Comment:
unsafe call of `.wordDataArray` (use a typeId like -1).

solution: `return typeName.labelDataArray[typeId]?.wordDataArray[0]?.str;`